### DISCUSSION TITLE
Data for Last F1 Race is available?

### DISCUSSION THREAD

**Discussion by JochemZ** [theOehrly/Fast-F1#136] [General]
I have this error, data ERGAST is updated, but i got message that it failed to load lap data.
I tried previous Race (Bahrein-2022) and this is working fine, so it is only the last race (Saudi - Jeddah)

core           INFO 	Loading data for Saudi Arabian Grand Prix - Race [v2.2.1]
api            INFO 	Using cached data for driver_info
api            INFO 	Using cached data for timing_data
api            INFO 	Using cached data for timing_app_data
core           INFO 	Processing timing data...
api            INFO 	Using cached data for session_status_data
core        WARNING 	Failed to load lap data!

**Comment by ezkomoly** [comment]
theOehrly released a fix today for this issue. 
It's caused by Yuki Tsunoda being in the data list, unlike Mick.
See #135 for more information, or just install the new update if you are not interested.

**Comment by theOehrly** [comment]
The release is out now. I accidentally closed the other issue a bit early. Please update to v2.2.2

**Reply by JochemZ** [reply]
thnx!
